fl4health/clients/apfl_client.py

```diff
@@ -24,7 +25,36 @@ def __init__(
         metric_meter_type: MetricMeterType = MetricMeterType.AVERAGE,
         checkpointer: Optional[TorchCheckpointer] = None,
     ) -> None:
-        super().__init__(data_path, metrics, device, loss_meter_type, metric_meter_type, checkpointer)
+        super(BasicClient, self).__init__(data_path, device)
```

Comment:
Rather than having the user construct MetricMeterManagers, we could have them just pass a set of Meter objects and have the constructor run a function called `setup_metric_meter_managers(meters: Sequence[Meters])`. By default it runs the setup in BasicClient and in APFL, we would just override the function. If you like the idea of constructing MetricMeterManagers though, that would work too.

Reply:
Yeah, that is also another option to consider! I will see what ends up being the cleanest and go with that. Will focus on this in the next PR